tests/helpers.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+from jenkins_job_wrecker.cli import get_xml_root, root_to_yaml
+import yaml
+
+def compare_jjb_output(fixtures_path, name, job_name):
+    test_filename = os.path.join(fixtures_path, name + '.xml')
+    expected_yml_filename = os.path.join(fixtures_path, name + '.yml')
+    root = get_xml_root(filename=test_filename)
+    actual_yml = root_to_yaml(root, job_name)
+
+    with open(expected_yml_filename) as f:
+        expected_yml = f.read()
+
+    expected_yml = yaml.dump(yaml.load(expected_yml, Loader=yaml.FullLoader))
```

Comment:
why do we need this step?

Reply:
I think I've answered myself - just to make sure the formatting is exactly the same on both sides? It's worth putting a comment here if the case.

Anyway, shouldn't the expected YAML file be already in the "right" formatting?

Reply:
This is done in order to sort the keys so that I would not have to each time manually on yaml files.

Reply:
So a comment would be useful for future reference.